cmd/juju/cloud/updatecredential.go

```diff
@@ -17,31 +17,31 @@ import (
 )
 
 var usageUpdateCredentialSummary = `
-Updates a controller credential for a cloud.`[1:]
+Updates a remote credential.`[1:]
 
 var usageUpdateCredentialDetails = `
-Cloud credentials for controller are used for model operations and manipulations.
-Since it is common to have long-running models, it is also common to 
-have these cloud credentials become invalid during models' lifetime.
-When this happens, a user must update the cloud credential that 
-a model was created with to the new and valid details on controller.
-
-This command allows to update an existing, already-stored, named,
-cloud-specific controller credential.
-
-NOTE: 
-This is the only command that will allow you to manipulate cloud
-credential for a controller. 
-All other credential related commands, such as 
-` + "`add-credential`" + `, ` + "`remove-credential`" + ` and  ` + "`credentials`" + ` 
-deal with credentials stored locally on the client not on the controller.
+This command updates a credential cached on a controller. It does this by
+uploading the identically named local credential and validating it against the
+cloud. Use the ` + "`credentials`" + ` and ` + "`show-credential`" + ` commands to view local and
+remote credentials respectively.
+
+This command does not affect what models the updated credential may be related
```

Comment:
I do not see why this needs to be mentioned here?

Reply:
I wanted to ensure the user that when they "update a credential" that any other settings related to that credential are not affected. Referring to `set-credential` is just an extra.

Reply:
Sure, but this feels like a general flow and is the concern outside of this page.. It's kind of like saying that add-model or deploy is preceded by 'bootstrap'...

It would suffice to say that the credential should be known to juju locally via either 'add-credential' or 'add-credential --replace'... 

Again, it feels like a lot of effort for the area that will have to be re-worded yet again shortly once the new changes to this CRUD commands are in.